libs/ui/common/src/lib/button/universal-button.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactNode } from "react";
+
+export function UniversalButton({
+	onClick,
+	title,
+	additionalClassNames,
+	children
+}: {
+	onClick: (event?: React.MouseEvent<HTMLButtonElement>) => void;
+	title?: string;
+	additionalClassNames?: string;
+	children: ReactNode;
+}) {
+	return (
+		<button
+			type="button"
+			className={`border-1 border-gray-150 inline-flex items-center justify-center rounded-md border bg-white px-4 py-2 font-medium text-white hover:bg-gray-100
+			${additionalClassNames || ""}`}
+			onClick={event => onClick(event)}
+			title={title}
+		>
+			{children}
+		</button>
+	);
+}
```

Comment:
wirklich kein guter Komponenten name. 

was ist denn ein universal button ? 

wir haben layout buttons, zB Primary, Secondary button etc. Aber was ein universal button ist weiß ich nicht. generell hat er eben so viele props wie eine normale <button> komponente. 


Entweder die kompnente ganz löschen, oder ein passenderen layout namen finden. Nennen wir ihn mal temporär `GreyBoarderButton`

```suggestion


export function GreyBoarderButton(props: PropsWithChildren<ButtonHTMLAttributes<HTMLButtonElement>>) {
	return (
		<button
			{...props}
			className="border-1 border-gray-150 inline-flex items-center justify-center rounded-md border bg-white px-4 py-2 font-medium text-white hover:bg-gray-100"
		>
			{props.children}
		</button>
	);
}
```

Reply:
Habe ich so übernommen.